### PR TITLE
🩹 [QAA-304] Adding check for operation history when adding an account

### PR DESCRIPTION
**Pull request by VicAlbr**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Adding minor check on Add account

### ❓ Context

- **JIRA or GitHub link**: [https://ledgerhq.atlassian.net/jira/software/c/projects/QAA/boards/887?selectedIssue=QAA-304](QAA-304)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-desktop/tests/page/portfolio.page.ts [hunk 1]
```diff
@@ -9,14 +9,17 @@ export class PortfolioPage extends AppPage {
...
   private marketPerformanceWidget = this.page.getByTestId("market-performance-widget");
   private swapButton = this.marketPerformanceWidget.getByRole("button", { name: "Swap" });
   private buyButton = this.marketPerformanceWidget.getByRole("button", { name: "Buy" });
   private assetAllocationTitle = this.page.getByText("Asset allocation");
   private trendTitle = this.marketPerformanceWidget.getByText("1W trend");
   private assetRowElements = this.page.locator("[data-testid^='asset-row-']");
   private showAllButton = this.page.getByText("Show all");
+  private showMoreButton = this.page.getByText("Show more");
   private assetRow = (asset: string) => this.page.getByTestId(`asset-row-${asset.toLowerCase()}`);
+  private operationRows = this.page.locator("[data-testid^='operation-row-']");
```
Comment (bharamboure-ledger) [inline apps/ledger-live-desktop/tests/page/portfolio.page.ts:22]: could you replace by `locator` to `getByTestId` please?

File: apps/ledger-live-desktop/tests/specs/speculos/add.account.spec.ts [hunk 2]
```diff
@@ -50,6 +50,7 @@ for (const currency of currencies) {
         if (currency.currency.name !== Currency.TON.name) {
           await app.layout.expectBalanceVisibility();
         }
+        await app.portfolio.opperationHistory();
```
Comment (bharamboure-ledger) [inline apps/ledger-live-desktop/tests/specs/speculos/add.account.spec.ts:53]: small typo 2 "p"

File: apps/ledger-live-desktop/tests/page/portfolio.page.ts [hunk 3]
```diff
@@ -81,9 +84,24 @@ export class PortfolioPage extends AppPage {
...
 
+  @step("Scroll to operations")
   async scrollToOperations() {
     await this.page.waitForTimeout(500);
     const operationList = this.page.locator("id=operation-list");
     await operationList.scrollIntoViewIfNeeded();
   }
+
+  @step("check operation history")
+  async opperationHistory() {
```
Comment (bharamboure-ledger) [inline apps/ledger-live-desktop/tests/page/portfolio.page.ts:95]: could you rename method to something like `checkOperationsHistory` please?

File: apps/ledger-live-desktop/tests/page/portfolio.page.ts [hunk 4]
```diff
@@ -81,9 +84,24 @@ export class PortfolioPage extends AppPage {
     await assetRowLocator.click();
   }
 
+  @step("Scroll to operations")
   async scrollToOperations() {
     await this.page.waitForTimeout(500);
     const operationList = this.page.locator("id=operation-list");
```
Comment (bharamboure-ledger) [inline apps/ledger-live-desktop/tests/page/portfolio.page.ts:90]: might make sense to use [this locator](https://github.com/LedgerHQ/ledger-live/pull/8168/files#diff-2ebb39b2b4d0534aaa0564f39f516ec7b8762351e4f5db726d952331bb1f820aR12)

**Review by bharamboure-ledger — APPROVED** [review]

**Review by LucasWerey — APPROVED** [review]

**Pull request merged**